utility_scripts/QuotaAudit.py

```diff
@@ -0,0 +1,105 @@
+#   Copyright 2017 Massachusetts Open Cloud
```

Comment:
General note, this PR is failing the CI checks because the file does not meet the PEP8 style guide.  If you look under where it says "All checks have failed" on the PR, click the "Details" link and it will show you the output from Travis CI.  It's mostly stuff like white space at the end of the lines, etc.

You can install the tool `flake8` to check your files locally - just run it from the command line on a given file like this:
`$ flake8 file-to-check.py`

There are tools to automate some of the tedious whitespace fixes, etc., but I suggest fixing this file manually just to get a sense of what the different style requirements are.

The PEP8 guidelines are here, if there are errors you don't understand, or you can ask me:
https://www.python.org/dev/peps/pep-0008/